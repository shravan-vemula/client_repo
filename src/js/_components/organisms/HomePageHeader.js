import React from "react";
import { makeStyles } from "@material-ui/core/styles";
import { AppBar, Toolbar } from "@material-ui/core";
import Title from "js/_components/atoms/Title";
import HeaderIcons from "js/_components/molecules/HeaderIcons";

const useStyles = makeStyles((theme) => ({
 
  title: {
    flexGrow: 1,
  },
}));

export default function HomePageHeader() {
  const classes = useStyles();

  return (
    <div className={classes.root}>
      <AppBar position="fixed" color="inherit">
        <Toolbar>
          <Title />
          <HeaderIcons />
        </Toolbar>
      </AppBar>
    </div>
  );
}
