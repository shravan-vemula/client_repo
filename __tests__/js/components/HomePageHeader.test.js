import React from "react";
import { cleanup } from "@testing-library/react";
import HomePageHeader from "js/_components/organisms/HomePageHeader";

afterEach(cleanup);

describe("HomePageHeader Component", () => {
  it("renders correctly", () => {
    const wrapper = shallow(<HomePageHeader />);
    expect(wrapper).toMatchSnapshot();
  });
});
