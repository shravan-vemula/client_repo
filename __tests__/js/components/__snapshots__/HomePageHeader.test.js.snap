// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePageHeader Component renders correctly 1`] = `ShallowWrapper {}`;
